sphinx_gallery/tests/test_full.py

```diff
@@ -83,7 +83,14 @@ def _sphinx_app(tmpdir_factory, buildername):
     src_dir = Path(__file__).parent / "tinybuild"
 
     def ignore(src, names):
-        return ("_build", "gen_modules", "auto_examples")
+        return (
+            "_build",
+            "gen_modules",
+            "auto_examples",
+            "auto_examples_README_header",
+            "auto_examples_rst_index",
+            "auto_examples_with_rst",
```

Comment:
Noticed these were missing, doesn't affect tests but I think if someone is running locally, prevents test copying over previously generated gallery files